src/test-utils/dom/form-field/index.ts

```diff
@@ -26,6 +26,10 @@ export default class FormFieldWrapper extends ComponentWrapper<HTMLElement> {
     return this.find(`:scope > .${styles.hints} .${styles.error__message}`);
   }
 
+  findErrorIconWrapper(): ElementWrapper | null {
```

Comment:
I'm not sure adding a test utility to find the error icon is necessary. I don't really see customers using this in their test code.

Reply:
Removed 👍